src/types/output_damage.zig

```diff
@@ -0,0 +1,53 @@
+const wlr = @import("../wlroots.zig");
+
+const pixman = @import("pixman");
+
+const wayland = @import("wayland");
+const wl = wayland.server.wl;
+
+pub const OutputDamage = extern struct {
+    output: *wlr.Output,
+    max_rects: i32,
+
+    current: pixman.Region32,
+
+    // See WLR_OUTPUT_DAMAGE_PREVIOUS_LEN
```

Comment:
I'm unsure about this. In wlroots this was a define.

I think just hardcoding the value is fine because it can not be overwritten in wlroots either. It ist just defined in the header.

Reply:
We don't even need the comment here. The only reason for such defines is because C has a weaker type system and arrays are just pointers and so you need a define to eliminate magic numbers when iterating over them for example.

Reply:
That makes sense. Thanks for the explanation. The comment is removed now.